packages/cli-tool/src/template/initialize-cra-app.ts

```diff
@@ -1,10 +1,16 @@
 import runCommand from '../helpers/child-process';
 
+const NIMBLE_CRA_TEMPLATE = '@nimblehq';
```

Comment:
Maybe `DEFAULT_CRA_TEMPLATE`? Since it is the default value when the template is not provided.

Reply:
Good idea! bb5c58b ✅